pkgs/by-name/mo/mouseless/package.nix

```
@@ -0,0 +1,37 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+
+let
+  version = "0.2.0";
+in
+buildGoModule {
+  pname = "mouseless";
+  inherit version;
+
+  vendorHash = "sha256-2q7L9BVcAaT4h/vUcNjVc5nOAFnb4J3WabcEGxI+hsA=";
+
+  src = fetchFromGitHub {
+    owner = "jbensmann";
+    repo = "mouseless";
+    tag = "v${version}";
```

Generally the preferred way to do this is with finalAttrs (see #315337), but you welcome to keep it like this if that's what you prefer.
```suggestion
buildGoModule (finalAttrs: {
  pname = "mouseless";
  version = "0.2.0";
  
  vendorHash = "sha256-2q7L9BVcAaT4h/vUcNjVc5nOAFnb4J3WabcEGxI+hsA=";

  src = fetchFromGitHub {
    owner = "jbensmann";
    repo = "mouseless";
    tag = "v${finalAttrs.version}";
```